src/pages/blog.jsx

```diff
@@ -0,0 +1,42 @@
+import React, {Fragment} from 'react';
+import { client } from '../lib/client';
+
+import Header from './components-global/header';
+import Footer from './components-global/footer';
+
+const Blog = ({blogs}) => {
+
+  return (
+    <Fragment>
+      <Header/>
+      <main>
+        {blogs?.map((blog) => {
+          return (
+            <div key={blog.id}>
+              <br/>
```

Comment:
@KennedyNvsf we need to make this spacing with proper CSS margins, I can look into it later if you don't know which element to target, but for now please remove these extra <br />'s